x/revenue/genesis.go

```diff
@@ -1,6 +1,7 @@
 package revenue
 
 import (
+	"cosmossdk.io/errors"
```

Comment:
Nit: refactor to be consistent w/codebase
```suggestion
	errorsmod "cosmossdk.io/errors"
```